deployment/keystone/changeset/add_capabilities_test.go

```diff
@@ -76,6 +79,124 @@ func TestAddCapabilities(t *testing.T) {
 	})
 }
 
+func TestAddCapabilitiesRequest_Validate_WriterCapability(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name          string
+		req           func(wrapper test.EnvWrapper) (*changeset.AddCapabilitiesRequest, error)
+		expectedError error
+	}{
+		{
+			name: "valid request with chain ID on capability name",
+			req: func(te test.EnvWrapper) (*changeset.AddCapabilitiesRequest, error) {
+				chainID, err := chainselectors.GetChainIDFromSelector(chainselectors.TEST_90000001.Selector)
+				if err != nil {
+					return nil, err
+				}
+				return &changeset.AddCapabilitiesRequest{
+					RegistryChainSel: te.RegistrySelector,
+					Capabilities:     []kcr.CapabilitiesRegistryCapability{{LabelledName: fmt.Sprintf("%s%s", changeset.CapabilityTypeTargetNamePrefix, chainID), Version: "1.0.0", CapabilityType: changeset.CapabilityTypeTarget}},
+					RegistryRef:      te.CapabilityRegistryAddressRef(),
+				}, nil
+			},
+			expectedError: nil,
+		},
+		// Cannot test this since `chainselectors.ChainIdFromName()` uses the chains from the `.yaml` files,
```

Comment:
See:

- https://github.com/smartcontractkit/chain-selectors/blob/main/evm.go#L103
- https://github.com/smartcontractkit/chain-selectors/blob/main/test_selectors.yml#L13

The test env generates chains from `90000001` on.

cc: @krehermann